### DISCUSSION TITLE
What insights would you like to see?

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/feedback#3] [General feedback or Bugs]
## Comment below with requests and feedback
OpenSauced transforms git commits into insights. We would love to know what insights you would like to see from repositories. 

![hacktoberfest](https://user-images.githubusercontent.com/5713670/194442266-6320b65c-ab29-475e-94e7-56f8ca73ea1f.png)

Our initial offering provided insights for Hacktoberfest. Do you know of any other events or hackathons that would like to get insights. 

## More Questions

- When you open the GitHub Insights tab in a repo, what data do you expect to see there?
- How do you encourage contributions to your projects?
- When does your community hang out?

**Comment by haastrupea** [comment]
Hi Everyone!
I registered to participate in the #100DaysOfOSS challenge and I got a coupon for a whole year.

I am in Nigeria, my local Debit card got declined and as a result could not use the coupon.

It would be great if a credit/debit card is not a requirement when a coupon giving 100% discount is applied.

**Reply by bdougie** [reply]
This is a little off topic for this thread. Do you mind creating a new thread?

**Reply by haastrupea** [reply]
on it
